package.json

```diff
@@ -8,10 +8,10 @@
     "node": ">=8.10.0"
   },
   "repository": "googleapis/nodejs-kms",
-  "main": "src/index.js",
+  "main": "build/src/index.js",
   "files": [
-    "protos",
-    "src",
+    "build/protos",
+    "build/src",
     "AUTHORS",
```

Comment:
While you're in here, can you drop `AUTHORS` and `COPYING`?

Reply:
sure, thanks